src/Tickets/Seating/app/frontend/ticketsBlock/index.js

```diff
@@ -14,7 +14,7 @@ import { TicketRow } from './ticket-row';
 import { localizedData } from './localized-data';
 import { formatWithCurrency } from '../../currency';
 import { getCheckoutHandlerForProvider } from './checkout-handlers';
-import { start as startTimer, reset as resetTimer } from '@tec/tickets/seating/frontend/session';
+import { start as startTimer, reset as resetTimer } from '@tec/tickets/seating/frontend/session'; // eslint-disable-line
```

Comment:
Why disable this line?

Reply:
@bordoni the linter doesn't know how to resolve the `@tec` namespace, so it triggers an error.